Dockerfile

```diff
@@ -1,39 +1,26 @@
-FROM java:7
+FROM java:8-jdk
 MAINTAINER Gabe Conradi <gabe@tumblr.com>
 
 RUN apt-get update && apt-get install -y zip unzip && rm -r /var/lib/apt/lists/*
 
-RUN useradd -Ur -d /opt/collins collins
-RUN for dir in /build /build/collins /var/log/collins /var/run/collins; do mkdir $dir; chown collins $dir; done
-
-WORKDIR /build
-# get Play, Collins, build, and deploy it to /opt/collins
 COPY . /build/collins
-RUN wget -q http://downloads.typesafe.com/typesafe-activator/1.3.6/typesafe-activator-1.3.6-minimal.zip -O /build/typesafe-activator-1.3.6-minimal.zip && \
-    unzip -q ./typesafe-activator-1.3.6-minimal.zip && \
+RUN cd /build && \
+    export ACTIVATOR_VERSION=1.3.7 && \
```

Comment:
Doesnt matter. its only within the context of this `RUN` command, so nothing leaks out into the built image.